account_credit_control/line.py

```diff
@@ -130,7 +130,7 @@ class CreditControlLine(models.Model):
                                 store=True,
                                 readonly=True)
 
-    level = fields.Integer('credit.control.policy.level',
+    level = fields.Integer(string='Policy Level',
```

Comment:
Maybe we should add it to the translation template, or add Transifex Travis job...